src/common/detectTerminalShell.c

```diff
@@ -232,7 +232,6 @@ const FFTerminalShellResult* ffDetectTerminalShell(FFinstance* instance)
 
     ffStrbufInit(&result.terminalProcessName);
     ffStrbufInit(&result.terminalExe);
-    result.terminalExeName = result.terminalExe.chars;
```

Comment:
I don't think this should be removed. I expect `result.terminalExeName` to always be a valid c-string. At this point it will _always_ point to the `'\0'` char of `result.terminalExe`. So setting it here can't really be the problem.